docs/content/build-instructions.md

```diff
@@ -42,6 +42,7 @@ Required packages:
 
 Optional packages (e.g., for language-specific support or building documentation):
   - default-jdk
+  - libjchart2d-java
```

Comment:
Because of the context here in the section title "(e.g., for language-specific support or building documentation)" I'd like this to indicate it will be built if the package is not installed such that readers don't conclude they must install this package to enable this functionality.  Similar to:

```suggestion
  - libjchart2d-java # note: if not installed, jchart2d will be built from source
```